com.arc.embeddedcdt/META-INF/MANIFEST.MF

```diff
@@ -23,14 +23,14 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.core.variables,
  org.eclipse.cdt.launch;bundle-version="7.1.0",
  org.eclipse.tm.terminal.control,
- org.eclipse.tm.terminal.connector.serial,
  org.eclipse.tm.terminal.view.ui,
  org.eclipse.tm.terminal.view.core,
  org.eclipse.cdt.dsf.gdb.ui,
  org.eclipse.cdt.dsf,
  org.eclipse.cdt.dsf.gdb,
- org.eclipse.cdt.launch.remote
+ org.eclipse.cdt.launch.remote,
+ org.eclipse.tm.terminal.connector.cdtserial,
+ org.eclipse.cdt.native.serial;bundle-version="1.1.0"
```

Comment:
Remove exact version specification - it is not used in most of the case in our plugins.